firebase_admin/_auth_utils.py

```diff
@@ -328,6 +328,15 @@ def __init__(self, message, cause, http_response):
         exceptions.InvalidArgumentError.__init__(self, message, cause, http_response)
 
 
+class InvalidHostingLinkDomainError(exceptions.InvalidArgumentError):
+        """Hosting link domain in ActionCodeSettings is not authorized."""
```

Comment:
Can you update both msgs to be
"Hosting link domain in ActionCodeSettings is not authorized or is not owned by the current project."? to be more aligned with node SDK.  See: https://github.com/firebase/firebase-admin-node/blob/26b884f70300ef8e3213dfbbd72351ce7f68c695/src/utils/error.ts#L510

Reply:
Certainly, appreciate you putting eyes on this. Updated in https://github.com/firebase/firebase-admin-python/pull/884/commits/bc7fcfcf7f052b9c569fc0cf17c1bf596db73804.